### PR TITLE
Improve Intl.DurationFormat.format examples

### PR DESCRIPTION
**Pull request by ben-allen**
…normative changes related to fractionalDigits option, clarify usage in general

### Description

* Updated examples to reflect https://github.com/tc39/proposal-intl-duration-format/pull/150
* Made existing examples much more comprehensive/accurate

### Motivation

Recent normative changes have altered the behaviour of `fractionalDigits. Intl.DurationFormat.format() now defaults to displaying however many digits are necessary to display all components, rather than defaulting to 0 fractional digits displayed.

### Related issues and pull requests

Relates to https://github.com/tc39/proposal-intl-duration-format/pull/150

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md [hunk 1]
```diff
@@ -64,42 +64,159 @@ const duration = {
...
+    <tr>
+      <td>
+        milliseconds, microseconds, nanoseconds
+      </td>
+      <td>
+        "long", "short", "narrow", "numeric"
+      </td>
+    </tr>
+  </tbody>
+</table>
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md:119]: No need for this table; it's already documented on https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DurationFormat/DurationFormat.

File: files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md [hunk 2]
```diff
@@ -42,15 +42,15 @@ const duration = {
 
 // Without options, style defaults to "short"
 new Intl.DurationFormat("en").format(duration);
-// "1 yr, 2 mths, 3 wks, 3 days, 4 hr, 5 min, 6 sec, 7 ms, 8 μs, 9 ns"
+// => "1 yr, 2 mths, 3 wks, 3 days, 4 hr, 5 min, 6 sec, 7 ms, 8 μs, 9 ns"
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md:45]: These arrows aren't recommended by our style guide; their absence is intentional.

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Hi @ben-allen thanks for the PR, but as a convention for `Intl` docs, the `format` pages are usually brief, and information about constructor options are included in the constructor page.

**Comment by ben-allen** [comment]
Updated to remove most of the examples, redundant explanations, and => in comments

File: files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md [hunk 3]
```diff
@@ -42,18 +44,10 @@ const duration = {
...
 new Intl.DurationFormat("en").format(duration);
-// "1 yr, 2 mths, 3 wks, 3 days, 4 hr, 5 min, 6 sec, 7 ms, 8 μs, 9 ns"
-
-// With style set to "long"
-new Intl.DurationFormat("en", { style: "long" }).format(duration);
-// "1 year, 2 months, 3 weeks, 3 days, 4 hours, 5 minutes, 6 seconds, 7 milliseconds, 8 microseconds, 9 nanoseconds"
-
-// With style set to "narrow"
-new Intl.DurationFormat("en", { style: "narrow" }).format(duration);
-// "1y 2mo 3w 3d 4h 5m 6s 7ms 8μs 9ns"
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/durationformat/format/index.md:53]: Curious why these are removed?

**Review by Josh-Cena — COMMENTED** [review]
@ben-allen One remaining question (https://github.com/mdn/content/pull/28708#discussion_r1303273272) and then we can merge this. I think it's valuable to have comparisons of different styles under the en locale.

**Review by Josh-Cena — APPROVED** [review]
LGTM. Thanks for the update!

**Pull request merged**